docs/authoring/servertaskauthoring.md

```diff
@@ -0,0 +1,77 @@
+    - EndpointId: - Specified endpointId details used while triggering below mentioned &#39;EndpointUrl&#39;. This input is optional
+    - Endpoint URL: - Http Url to trigger on task execution
+    - Method: - Http method like PUT, GET, POST, PATCH, DELETE, OPTIONS, HEAD, TRACE.
+    - Body: - Body of http request
+    - Headers: - headers of Http request. Header should be in JSON format.
+    - WaitForCompletion: -  you can specify true or false. Default value false. If you specify &#39;true&#39;, server waits for TaskCompleted event happen from external. If task event not comes within task timeout, task will be canceled. This input is optional
+    - Expression: - Expression Criteria which defines when to pass the task. No expression means response content does not influence the result. Example: - For response {&quot;status&quot;: &quot;successful&quot;}, the expression can be eq(root[&#39;status&#39;], &#39;successful&#39;). More Information[https://go.microsoft.com/fwlink/?linkid=842996](https://go.microsoft.com/fwlink/?linkid=842996)
+
+    **Cancel**: Configures the HTTP payload and properties for the Cancel event raised from VSTS to the external service. This property is optional.  You can configure these properties if you want call external service on user cancels the task. Properties are same as 'Execute' properties defined above.
+
+- **ServiceBus**
+This enables you to post a message to azure service bus queue. It takes 3 inputs.
+
+    **Events:** Same set of events mentioned above are supported in ServerBus also.
+    
+    **Execute:** Configures the servicebus message payload and properties for the Execute event raised from VSTS to the cloud service. ServiceBus handler has following properties.
+    - EndpointId: - EndpointId details used while publishing the message
+    - ConnectionString: - Azure service bus connection string
+    - ServiceBusQueueName: - Queue name to publish message
+    - MessageBody: - Message body. Message body should be JSON format
+    - MessageProperties: - message properties. like Content-Type:\application/json&quot;
+    - CertificateString: -  used to encrypt the message. This input is optional
+    - SignaturePropertyKey: - used to encrypt the message. This input is optional
+  
+    **Cancel**:  configures the servicebus message payload and properties for the Cancel event raised from VSTS to the cloud service. This property is optional. Properties are same as 'Execute' properties defined above.
+
+Task inputs, all well know system variables, user defined variables and endpoint variables can use in 'execute' properties. Example '"Body": "$method"', here 'method' is task input.
+     
+Let us understand few concepts/tips &amp; tricks.
+
+- A task which runs on agent less phase can&#39;t run on agent phase or deployment group.
+- Async/Sync mode.
```

Comment:
What is the purpose of this note? I don't see a mechanism in the HTTP handler documentation for controlling sync or async, and I'm not sure we need to provide this type of control.